> /Users/andrewmiotke/GitHub/220-Advanced-Summer-2019/students/AndrewMiotke/lesson02/assignment/src/charges_calc.py(3)<module>()
-> '''
(Pdb) j 64
> /Users/andrewmiotke/GitHub/220-Advanced-Summer-2019/students/AndrewMiotke/lesson02/assignment/src/charges_calc.py(64)<module>()
-> data = load_rentals_file(args.input)
(Pdb) s
NameError: name 'load_rentals_file' is not defined
> /Users/andrewmiotke/GitHub/220-Advanced-Summer-2019/students/AndrewMiotke/lesson02/assignment/src/charges_calc.py(64)<module>()
-> data = load_rentals_file(args.input)
(Pdb) s source.json
--Return--
> /Users/andrewmiotke/GitHub/220-Advanced-Summer-2019/students/AndrewMiotke/lesson02/assignment/src/charges_calc.py(64)<module>()->None
-> data = load_rentals_file(args.input)
(Pdb) s
NameError: name 'load_rentals_file' is not defined
> <string>(1)<module>()->None
(Pdb) n
--Return--
> <string>(1)<module>()->None
(Pdb) n
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/pdb.py", line 1701, in main
    pdb._runscript(mainpyfile)
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/pdb.py", line 1570, in _runscript
    self.run(statement)
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/bdb.py", line 585, in run
    exec(cmd, globals, locals)
  File "<string>", line 1, in <module>
  File "/Users/andrewmiotke/GitHub/220-Advanced-Summer-2019/students/AndrewMiotke/lesson02/assignment/src/charges_calc.py", line 64, in <module>
    data = load_rentals_file(args.input)
NameError: name 'load_rentals_file' is not defined
Uncaught exception. Entering post mortem debugging
Running 'cont' or 'step' will restart the program
> /Users/andrewmiotke/GitHub/220-Advanced-Summer-2019/students/AndrewMiotke/lesson02/assignment/src/charges_calc.py(64)<module>()->None
-> data = load_rentals_file(args.input)
(Pdb)
